src/parseWithPointers.ts

```diff
@@ -12,30 +22,408 @@ export const parseWithPointers: SourceMapParser = <T>(value: string): IParserRes
 
   if (!value || !value.trim().length) return parsed;
 
-  const result = jsonSourceMap.parse(value);
-  parsed.data = result.data;
-  parsed.pointers = transformPointers(result.pointers);
+  const validations: IValidationResult[] = [];
 
-  return parsed;
+  return {
+    ...parse(value, validations, {
+      disallowComments: true,
```

Comment:
Actually, it was my own call - wanted to make sure we are more aligned with the old solution. I believe the discussion is still open - if you guys think it would make sense to support comments, we can change the flag.

Reply:
I think you made the right choice. If we export files we want to make sure they can be used in other tools not that sophisticated as Stoplight, therefore keeping it in this way is ok.